jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -209,8 +212,8 @@ public BuildConfiguration build() {
               targetImageCredentialHelperName,
               knownTargetRegistryCredentials,
               mainClass,
-              ImmutableList.copyOf(javaArguments),
-              ImmutableList.copyOf(jvmFlags),
+              copyToImmutableListWithoutNulls(javaArguments),
```

Comment:
Would this be worth applying within the setter, so that it can be documented on the javadoc?  And should we be filtering empty arguments?

Reply:
Yea, I think we should have the Builder setters take in Immutable* to begin with and do this null filtering on the Maven plugin side.